### PR TITLE
Add mac80211_hwsim configuration

### PR DESCRIPTION
**Pull request by ldotrg**
1. Add mac80211_hwsim demo configuration which include hostapd and wpa_supplicant.
    - Create a AP as a mock for "owl0 wifi"
    - In the future, "owl0" will replace the wpa_supplicant interface wlan0 
2. Fix verify.sh could not find common.sh file.

### CODE REVIEW

**Review by jserv — CHANGES_REQUESTED** [review]
Rebase the latest "main" branch

File: scripts/mac80211_hwsim_demo.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# mac80211_hwsim demo
```
Comment (jserv) [inline scripts/mac80211_hwsim_demo.md:1]: Rename this file to `scripts/README.md`

File: scripts/mac80211_hwsim_demo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Hostapd set up
+```bash=
```
Comment (jserv) [inline scripts/mac80211_hwsim_demo.md:3]: Annotate with `shell` rather than `bash=`.

File: scripts/verify.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 export DEVENVROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-source $DEVENVROOT/scripts/cmn.sh
+source $DEVENVROOT/scripts/common.sh
```
Comment (jserv) [inline scripts/verify.sh:3]: Send another pull request for this fix.

**Comment by jserv** [comment]
By the way, can you address the motivation of this pull request? e.g., why does it make sense to `vwifi` development?

File: scripts/wpa_supplicant.conf [hunk 4]
```diff
@@ -0,0 +1,14 @@
+ctrl_interface=/var/run/wpa_supplicant 
+network={
+    ssid="octobersky"
+    key_mgmt=NONE
+}
+
+# network={
```
Comment (jserv) [inline scripts/wpa_supplicant.conf:7]: Why did you leave unused item?

**Comment by ldotrg** [comment]
Abort this pull request.
When mock AP is ready, I will create another pull request.

**Pull request closed without merging**